### PR TITLE
Build openjdk8 mac

### PR DESCRIPTION
**Pull request by allandequeiroz**

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
     * [Build your own VM](virtual-machines/build_your_own_vm.md)
     * [Build your own light-weight VM](virtual-machines/build_your_own_lightweight_vm.md)
     * [Sharing host folder with guest VM](virtual-machines/sharing_host_folder_with_guest_vm.md)
+    * [OpenJDK 8 Vagrant Puppet ](virtual-machines/adoptjdk_puppet_vm.md)
```
Comment (neomatrix369) [inline en/SUMMARY.md]: White space near Puppet

File: pt/SUMMARY.md [hunk 2]
```diff
@@ -17,6 +17,8 @@
    * [Faça sua própria VM](virtual-machines/build_your_own_vm.md)
    * [Faça sua própria VM light-weight](virtual-machines/build_your_own_lightweight_vm.md)
    * [Compartindo arquivos locais com sua VM](virtual-machines/sharing_host_folder_with_guest_vm.md)
+   * [OpenJDK 8 Vagrant Puppet ](virtual-machines/adoptjdk_puppet_vm.md)
```
Comment (neomatrix369) [inline pt/SUMMARY.md]: White space

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+# Building OpenJDK8 on Mac, Inception approach.
+
+You can see another version of this turorial [here](http://allandequeiroz.io/2017/05/15/building-openjdk-8-on-mac-inception-approach)
+
+This is an alternative way to build OpenJDK8 on recent versions of OSx.
+
+
+Different of the OpenJDK 9 where you can use the most recent versions of XCode to build the JDK, OpenJDK 8 still using XCode4 to get the job done, if you're facing problems using xcode-select or --with-xcode-path flags to build it directly on your machine an alternative is using a VM with the appropriate configurations.
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: Different of the....  ==> Different than the....

Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: Try using backticks or triple-backticks to highlight keywords like names of programs or command-line flags.

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+In this tutorial, we're using VirtualBox 5.1.22, anything else related like XCode, JDK and freetype are already configured in the adopt-openjdk-jdk8u-build VM, so first step, downloads:
+
+[http://download.virtualbox.org/virtualbox/5.1.22/VirtualBox-5.1.22-115126-OSX.dmg](http://download.virtualbox.org/virtualbox/5.1.22/VirtualBox-5.1.22-115126-OSX.dmg)
+
+[https://goo.gl/Iw838n](https://goo.gl/Iw838n)
+
+# Throubleshooting
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: Troubleshooting

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+![](../../assets/1.png)
+
+Start again and keep pressing **F12**, I'll see the virtual machine "BIOS", select **Boot Manager** then **EFI Hard Drive** and **Continue**
+
+![](../../assets/2.png)
+
+![](../../assets/3.png)
+
+# Configurations ( One time needed )
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: One time needed ==> one off

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+![](../../assets/6.png)
+
+Using VirtualBox start **adopt-openjdk-jdk8u-build** VM
+
+
+### VM
+
+Open Finder inside the VM, you'll notice that your host machine is listed at the left side under **sharing**, click on the shared folder and use the login and password that you has just created on your host machine to get access.
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: on the left side

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+- login : adoptopenjdk
+- password : adoptopenjdk
+
+Now, to automount this sharing folder across restarts, go to **System Preferences -> Users & Groups**, find the adoptopenjdk user, go to Login Items tab and add the recently mounted sharing folder, It will be located at **/Volumes/AdoptOpenJDK**
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: shared folder (not sharing folder)

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+![](../../assets/7.png)
+
+To make it easy, you can created an alias from **/Volumes/AdoptOpenJDK** into **/Users/adoptopenjdk/dev**
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: into => to

File: en/virtual-machines/build_openjdk_jdk8u_virtual_box.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+**System Preferences -> Users & Groups**
+
+Add new account of type "Sharing Only"
+
+![](../../assets/4.png)
+
+To make easy, we're creating a user with the same name as the default one used inside the VM
+
+- login : adoptopenjdk
```
Comment (neomatrix369) [inline en/virtual-machines/build_openjdk_jdk8u_virtual_box.md]: backticks around this info

**Comment by neomatrix369** [comment]
@allandequeiroz merge conflicts and comment(s) pending

File: en/SUMMARY.md (hunk 1 above)
Comment (allandequeiroz) [inline en/SUMMARY.md]: Hi @neomatrix369 locally they are fine, I think is a GitHub issue to present it, take a look at the other ones above, all of them have the same spaces here, but actually, they don't.

File: pt/SUMMARY.md (hunk 2 above)
Comment (allandequeiroz) [inline pt/SUMMARY.md]: Same as above.

**Comment by neomatrix369** [comment]
@allandequeiroz the repo still have merge conflicts, can't merge without them been resolved

**Comment by allandequeiroz** [comment]
That bit "conflicts" I was expecting. I'll fix it tomorrow morning.

**Comment by neomatrix369** [comment]
Thanks @allandequeiroz please tweet and retweet about the MacOSX build page

**Comment by allandequeiroz** [comment]
No problem, anything else just let me know. I'll tweet there!

**Comment by allandequeiroz** [comment]
https://twitter.com/allandequeiroz/status/865877660295155712

**Pull request merged**